features.md

```diff
@@ -41,12 +41,50 @@
    
 * ### Working Offline
     <!--Explain a bigger picture of sync on/off feature and how to switch-->
+    The openMRS android client can also be sucessfully used in areas with limited connectivity. As you open the OpenMRS android client    the login screen appears. Here you may provide your login credentials and a location (e.g. "Registration Desk")
+    To use the client in the offline mode you may press the option just beside the word "online" to switch to the offline mode.
+    
+   ![OpenMRS Android Client Offline login screen][login]
+    
+    
+    [login]: assets/openmrs_android_client_offline_login.png
+
 
    1. Registering Patients
 
       <!-- Explain delayed identifier assignment and duplicate patients resolution  -->
+            After successfully logging in you will be presented with a dashboard from where you can register or view patients.
```

Comment:
Please remove the preceding spaces. Markdown is interpreting it as a code sample.